source/includes/entityref/_federaltaxform.md

```diff
@@ -0,0 +1,154 @@
+# FederalTaxForm
+
+This entity represents a tax form that is added to a candidate at the federal level.
+
+This will only be used in the user interface if the corporation setting "novoTaxInfoTab" is enabled
```

Comment:
Change to:
The FederalTaxForm entity is only available in the user interface if the **novoTaxInfoTab** corporation setting  is enabled.